src/sidebar/store/use-store.js

```diff
@@ -6,14 +6,14 @@ import shallowEqual from 'shallowequal';
 import warnOnce from '../../shared/warn-once';
 import { useService } from '../util/service-context';
 
-/**
- * @typedef {import("redux").Store} Store
- */
+/** @typedef {import("redux").Store} Store */
+
+/** @typedef {import("./index").SidebarStore} SidebarStore */
```

Comment:
I believe TS will still parse this correctly these two type imports were folded into a single `/** ... */` block.